app/controllers/api/v0/packages_controller.rb

```diff
@@ -0,0 +1,12 @@
+module Api
+  module V0
+    class PackagesController < ApplicationController
+      def index
+        sql = 'lower(name) like ?'
```

Comment:
I don't exactly understand why do we need to separate this logic in 3 lines if it can be done in one. In my opinion, it doesn't affect code readability.

Reply:
As I noticed usually different difficult queries are replaced with other classes (service-objects and query-objects) but there is a really simple case.

Reply:
So i have to disable rubocop for this case?